radiance_folder/model/README.md

```diff
@@ -0,0 +1,127 @@
+# Model folder
+
+Model folder includes all the geometry and material files that describes the Radiance
+model.
+
+```cmd
+└─model                :: model folder
+  ├───aperture         :: static apertures description
+  ├───aperture_group   :: apertures groups (AKA window groups)*
+  │   └───interior     :: interior aperture groups
+  ├───bsdf             :: in-model BSDF files and transmittance matrix files
+  ├───grid             :: sensor grids
+  ├───ies              :: electric lights description
+  ├───scene            :: static scene description
+  ├───scene_dynamic    :: dynamic scene description*
+  │   └───indoor       :: indoor dynamic scene description*
+  └───view             :: indoor and outdoor views
+```
+
+There is a `README.md` file in each sub-folder with more information about that
+sub-folder.
+
+# Folder structure in relation to Radiance studies
+
+This folder structure is designed to accommodate most of possible daylight workflows but
+it does not mean that all the files are necessary for all the simulations. For instance
+the separation of indoor aperture_group is only meaningful for 3 or 5-phase simulations
+where view matrix and daylight matrix calculation happens.
```

Comment:
Don't you mean that it's meaningful for any multi-phase study with dynamic apertures?  A 2-phase study with dynamic apertures still requires the separation, right?

Also, it's more consistent to say ` separation of interior aperture_group` instead of ` separation of indoor aperture_group`

Reply:
Yes. you are correct. I'll fix this. Thanks.